website/content/docs/enterprise/license/faq.mdx

```diff
@@ -65,8 +65,17 @@ If you are an existing HashiCorp enterprise customer you may contact your organi
 You can use `nomad license get` to retrieve information about the license, but not the license itself. 
 
 ## Q: What happens when a license expires?
-If a license expires after a server node joins the cluster, the existing Enterprise features will continue to operate until a grace period expires. 
-The grace period duration is 1 day, and upon the end of the grace period the Nomad servers will shut down. 
+
+As a Nomad Enterprise license approaches its expiration time, Nomad servers will
+periodically log a warning message about the approaching expiration.
+
+When the license expires, enterprise functionality will become limited. Only
+read operations on enterprise endpoints will be supported, and write operations
+will return an error.
+
+Note that if the server is restarted with an expired license, it will
+immediately stop. Clients are unaffected by expired licenses because clients do
+not read the license.
```

Comment:
@gulducat has pointed out that this doesn't explain how _terminated_ licenses work, which is different from expiration. The docs don't seem to cover this either, so maybe we should expand on the docs in this PR too?

Reply:
Following up on this: the only way to actually verify the difference between terminated and expired licenses is to issue a license, but we can't issue expired licenses. So I need to issue myself a license with a near-future expired date far-future terminated date and then wait a day for it to expire. I'll issue that license now and then test it on Monday before we wrap this up.